virttest/tests/unattended_install.py

```diff
@@ -1130,6 +1130,7 @@ def copy_images():
             from virttest import utils_test
             error_context.context("Copy image from NFS Server")
             utils_test.run_image_copy(test, params, env)
+            utils_test.update_qcow2_image_version(params)
```

Comment:
For all cases which call run_image_copy() will run update_qcow2_image_version().